pallets/schemas/src/lib.rs

```diff
@@ -304,6 +355,10 @@ pub mod pallet {
 		///
 		#[pallet::call_index(2)]
 		#[pallet::weight(T::WeightInfo::propose_to_create_schema(model.len() as u32))]
+		#[allow(deprecated)]
+		#[deprecated(
+			note = "please use `propose_to_create_schema_v2` since `propose_to_create_schema` has been deprecated."
```

Comment:
One of my concerns about this changeset is potential confusion over the use of v2 versus v3.  It's clear - to me, from reading this PR - that there is a v2 of the interface/extrinsics but the schema structure is v3.  Can we skip over "v2" and align the extrinsic names with the schema version?

Reply:
I think there are 2 versions that we talking about.

- Version of a specific extrinsic
- Storage Migration 

They are independent of each other and I'm not sure mixing them is a great idea but open to all suggestions. We can discuss it as a parking lot.